### DISCUSSION TITLE
Changelog for Android version ?

### DISCUSSION THREAD

**Discussion by Go-rom** [CollaboraOnline/online#7261] [Q&A] [answered]
Hi,

I just got an update of Collabora for Android on my Phone, that's great news !

About that, is anyone could tell me where I can find the changelog please ?

Thanks a lot

**Comment by Tex2002ans** [comment, marked as answer]
Hey @Go-rom , thanks for the question. :)

> I just got an update of Collabora for Android on my Phone, that's great news !

Yes, Android got a huge jump from 21.11 -> 23.05!

> About that, is anyone could tell me where I can find the changelog please ?

I don't believe there's an Android-specific changelog, but for major features, you can take a look at the:

- [22.05 Press Release](https://www.collaboraoffice.com/collabora-online-development-edition-22-05-brings-powerful-new-features-like-grammar-checks-sparklines-and-jumbo-spreadsheets/)
- [23.05 Press Release](https://www.collaboraoffice.com/releases-en/collabora-online-23-05/)

and for more detailed changes, see the:

- [22.05 Release Notes](https://www.collaboraoffice.com/code-22-05-release-notes/)
- [23.05 Release Notes](https://www.collaboraoffice.com/code-23-05-release-notes/)

Since there was a huge rebase, the Android+iOS versions got lots of those features/fixes as well...

Like much better Dark Mode.

- - -

**Side Note:** If you want the exact, technical code changes, you can always compare the Github release "Tags":

- https://github.com/CollaboraOnline/online/tags

For example, this compares:

- [23.05.4 vs. 23.05.5](https://github.com/CollaboraOnline/online/compare/cp-23.05.4-2...cp-23.05.5-2)

- - -

> Thanks a lot

You're welcome. :)

Also, the Collabora developers don't use Github's Q&A so much...

In the future, it would be better to ask this sort of question in the:

- [Collabora Online Forum](https://forum.collaboraonline.com/)
